File: Server/object/Club.js
class Club {
    constructor(name, description, location, camera, price, time, difficulties) {
        this.name = name; 
        this.description = description;
        this.location = location;
        this.camera = camera == "Forbiden" ? false : true;
        this.price = price;
        this.time = time;
        this.difficulties = difficulties;
        this.comments = [""];
        this.rate = 0.0;
    }
}

File: lib/models/club.dart
import 'package:review/review.dart';

enum Entrance {
  easy,
  medium,
  hard,
}

class Club {
  String name;
  String description;
  String location;
  bool camera;
  String price;
  Entrance entrance;
  String clubbingTime;
  double? rating;
  List<Review> reviews;

  Club({
    required this.name,
    required this.description,
    required this.location,
    required this.camera,
    required this.price,
    required this.entrance,
    required this.clubbingTime,
    required this.rating,
    required this.reviews,
  });
}
